src-front/src/pages/Lobby/Lobby.tsx

```diff
@@ -0,0 +1,91 @@
+import { useNavigate } from "react-router-dom";
+import styles from "./Lobby.module.css";
+import Card from "../../components/Card/Card";
+import { useEffect, useState } from "react";
+import { API } from "../../lib/api";
+
+// import { getEnv } from "../../environment";
+
+export default function Lobby() {
+  const navigate = useNavigate();
+  //const refLink = useRef<string>();
+  const [fightId, setFightID] = useState<string>("");
+  const [userId, setUserID] = useState<string>("");
+
+  useEffect(() => {
+    const initializeFight = async () => {
+      const request = await API.createFight([]);
+      const response = await request.json();
+      setFightID(response["fightId"]);
+    };
+
+    initializeFight();
+  }, []); // Empty dependency array means this effect runs once on mount
+
+  const createRoom = async () => {
+    const userIdInput = document.getElementById(
+      "debugUserId",
+    ) as HTMLInputElement;
+
+    if (userIdInput == null) {
+      console.log("Error while searching for input fields.");
+      return;
+    }
+    setUserID(userIdInput?.value);
+
+    // const request = await API.createFight([]);
```

Comment:
If this is dead code, it is better to remove it to avoid confusion in the future. Otherwise, put a comment explaining why we should keep it.

Reply:
I think we'll need this useState later when we have the userId accessible via the sessions infos.

Reply:
I meant `// const request = await API.createFight([]);`, is this needed?